### ISSUE TITLE
Add compatibility with compression-webpack-plugin 10.0.0

### ISSUE THREAD

**Issue by tagliala** [shakacode/shakapacker#118]
Hi,

I don't know if this can make for 6.3.0, but [compression-webpack-plugin](https://github.com/webpack-contrib/compression-webpack-plugin) 10.0 has just been released and now there is a warning if you try to use it

```
warning " > shakapacker@6.2.1" has incorrect peer dependency "compression-webpack-plugin@^9.0.0".
```

**Comment by tagliala** [comment]
Will be fixed by #117

**Comment by tomdracz** [comment]
Closed by https://github.com/shakacode/shakapacker/pull/117 Will be released in the next version
